script.sql

```diff
@@ -0,0 +1,83 @@
+CREATE TABLE pracownik (
+	id bigint  primary key auto_increment,
+    imie VARCHAR(30),
+    nazwisko VARCHAR(50),
+    wyplata DECIMAL,
+    data_urodzenia DATE,
+    stanowisko VARCHAR(50)
+);
+
+INSERT INTO pracownik(imie, nazwisko, wyplata, data_urodzenia, stanowisko)
+VALUES ('Jan','Kowalski',5000,'1994-02-19','Helpdesk Operator'),
+('Jan','Niezbedny',2000,'1990-01-22','Helpdesk Lead'),
+('Lukasz','Wozny',15000,'1988-09-18','DevOps'),
+('Patryk','Welna',25000,'1980-12-07','COBOL Developer'),
+('Grzegorz','Wejna',22000,'1982-09-19','JAVA Developer'),
+('Kamil','Gruszka',35000,'1991-11-12','Solidity Developer');
+
+SELECT * FROM pracownik ORDER BY(nazwisko);
+
+SELECT * FROM pracownik WHERE stanowisko='COBOL Developer';
+
+SELECT * FROM pracownik WHERE data_urodzenia <= 1991-01-01;
+
+UPDATE pracownik SET wyplata= wyplata + (wyplata * 10 / 100);
+
+DELETE FROM pracownik WHERE  pracownik.id =  (SELECT id FROM (SELECT id, data_urodzenia
+FROM pracownik
+WHERE data_urodzenia = (
+    SELECT MIN(data_urodzenia)
+    FROM pracownik
+)) AS id);
+
+DROP TABLE pracownik;
+DROP TABLE stanowisko;
+DROP TABLE adres;
+
+CREATE TABLE stanowisko (
+	id bigint  primary key auto_increment UNIQUE NOT NULL,
```

Comment:
raczej nie ma potrzeby pisania UNIQUE i NOT NULL - jak coś jest primary key, to z założenia musi być unikalne i nie może być nulem